webapp/components/drawer/Drawer.js

```diff
@@ -58,17 +59,50 @@ const DrawerItem = ({
   </Tooltip>
 );
 
-export default ({
+
+const drawerStyles = ({spacing}) => ({
```

Comment:
I see some new eslint errors, please make sure that you can see & fix them in your IDE